src/tasks/dailyItems.ts

```diff
@@ -148,6 +148,21 @@ export const DailyItemTasks: Task[] = [
         }
       },
     },
+    {
+      name: "2002 Mr. Store",
+      // eslint-disable-next-line libram/verify-constants
+      ready: () => have($item`2002 Mr. Store Catalog`),
+      completed: () =>
+        get("availableMrStore2002Credits", 0) > 0 && get("_2002MrStoreCreditsCollected", false),
```

Comment:
You want there to be 0 credits left for this to be completed, right?

```suggestion
        get("availableMrStore2002Credits", 0) === 0 && get("_2002MrStoreCreditsCollected", false),
```

Reply:
Oopsie